src/main/java/seedu/address/logic/commands/CommandResult.java

```diff
@@ -17,21 +19,25 @@
     /** The application should exit. */
     private final boolean exit;
 
+    /** Switch to the tab indicated by tabId */
+    private final Index tabId;
```

Comment:
Will it be confusing to have TAB_ID in Command.java and tabId in CommandResult.java? Maybe more descriptive name like tabIdToSwitchTo can help.